plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -719,8 +719,13 @@ def get_have(self, config):
             Authentication and Policy Server information.
         """
 
-        if config.get("authentication_policy_server") is not None:
-            self.get_have_authentication_policy_server(config).check_return_status()
+        authentication_policy_server = config.get("authentication_policy_server")
+        if authentication_policy_server is not None:
+            self.get_have_authentication_policy_server(authentication_policy_server).check_return_status()
+        else:
+            self.msg = "authentication_policy_server in config is missing in the playbook"
```

Comment:
self.msg = "The 'authentication_policy_server' is missing in the playbook configuration."